tests/visualization_tests/test_slice.py

```diff
@@ -75,6 +75,7 @@ def _create_study_mixture_category_types() -> Study:
     return study
 
 
+@pytest.mark.filterwarnings("ignore::DeprecationWarning")
```

Comment:
Is there some warning message in this test?
```suggestion
```

Reply:
Yes,  there were deprication warnings